test/coffeescript.coffee

```diff
@@ -30,7 +30,7 @@ describe 'coffeescript', ->
 				throw new MyError()
 			catch e
 
-			expect(e).to.have.deep.property('constructor.name', 'MyError')
+			expect(e.constructor.name).to.be.equal('MyError')
```

Comment:
@emirotin `deep` was correct for the chaijs version that was targeted - that was a breaking change of chaijs v4